src/main/java/at/hannibal2/skyhanni/features/misc/GhostCounter.kt

```diff
@@ -0,0 +1,326 @@
+        val counter = hidden?.ghostCounter ?: return
+        KILLS.set(counter.ghostKills)
+        SORROWCOUNT.set(counter.sorrowCount)
+        VOLTACOUNT.set(counter.voltaCount)
+        PLASMACOUNT.set(counter.plasmaCount)
+        GHOSTLYBOOTS.set(counter.ghoostlyBootsCount)
+        BAGOFCASH.set(counter.bagOfCashCount)
+        TOTALDROPS.set(counter.totalDrops)
+        GHOSTSINCESORROW.set(counter.ghostSinceSorrow)
+        TOTALMF.set(counter.totalMF)
+        SCAVENGERCOINS.set(counter.scavengerCoins)
+        MAXKILLCOMBO.set(counter.maxKillCombo)
+        hasScavengerTalisman = counter.hasScavengerTalisman
+        update()
+    }
+
+    private fun drawDisplay() = buildList<List<Any>> {
+        val value = when (SORROWCOUNT.get()) {
+            0.0 -> "0"
+            else -> "${((((KILLS.get() / SORROWCOUNT.get()) + Math.ulp(1.0)) * 100) / 100).roundToInt()}"
+        }
+        val mgc = when (TOTALDROPS.get()) {
+            0.0 -> "0"
+            else -> "${((((TOTALMF.get() / TOTALDROPS.get()) + Math.ulp(1.0)) * 100) / 100).roundToPrecision(2)}"
+        }
+
+        addAsSingletonList("§6Ghosts counter")
+        val list = mapOf(
+                "Gosts Killed" to Pair(KILLS.getInt(), session.kills.roundToInt()),
+                "Sorrows" to Pair(SORROWCOUNT.getInt(), session.sorrow.roundToInt()),
+                "Ghost Since Sorrow" to Pair(GHOSTSINCESORROW.getInt(), ""),
+                "Ghosts/Sorrow" to Pair(value, ""),
+                "Volta" to Pair(VOLTACOUNT.getInt(), session.volta.roundToInt()),
+                "Plasma" to Pair(PLASMACOUNT.getInt(), session.plasma.roundToInt()),
+                "Ghostly Boots" to Pair(GHOSTLYBOOTS.getInt(), session.boots.roundToInt()),
+                "Bag Of Cash" to Pair(BAGOFCASH.getInt(), session.bag.roundToInt()),
+                "Avg Magic Find" to Pair(mgc, ""),
+                "Scavenger Coins" to Pair(SCAVENGERCOINS.getInt(), session.scavenger.roundToInt()),
+                "Kill Combo" to Pair(KILLCOMBO.getInt(), ""),
+                "Highest Kill Combo" to Pair(MAXKILLCOMBO.getInt(), session.maxcombo.roundToInt()),
+        )
+
+        for ((text, value) in list) {
+            val (total, session) = value
+            val se = if(session == "") "" else "($session)"
+            addAsSingletonList(config.formatText.replace("&", "§")
+                    .replace("%text%", text)
+                    .replace("%value%", "$total")
+                    .replace("%session%", se))
+        }
+    }
+
+    @SubscribeEvent
+    fun onInventoryOpen(event: InventoryOpenEvent) {
+        if (!isEnabled()) return
+        val inventoryName = event.inventoryName
+        if (inventoryName.startsWith("Accessory Bag")) {
+            val stacks = event.inventoryItems
+            for ((_, stack) in stacks) {
+                val name = stack.name ?: return
+                if (name.removeColor().contains("Scavenger Talisman")) {
```

Comment:
Maybe save this in the profile specific config

Reply:
I'm using PurseChangeEvent so i don't need that anymore